### DISCUSSION TITLE
Select Field Triggers Modal Even When Placeholder is Selected

### DISCUSSION THREAD

**Discussion by pepitovsevilla** [chakra-ui/chakra-ui#8794] [Q&A] [answered]
**Description**
I'm working with a Select field that opens a modal when an option is selected. The modal header displays the text of the selected option. However, when I select the placeholder from the Select box, it also triggers the modal and displays an empty string "" in the ModalHeader.

Is it normal behavior for onChange to be called when the placeholder is selected? If so, what's the best way to prevent the modal from opening when the placeholder is chosen?

**Link to Reproduction**
https://codesandbox.io/p/devbox/6s5h46

**Steps to reproduce**
Go to https://codesandbox.io/p/devbox/6s5h46
Click on Select box
Select any option
Close the modal
Click on the Select box again
Select the placeholder

**Chakra UI Version**
2.6.1

**Browser**
Chrome 128.0.6613.113

**Operating System**
 macOS

**Additional Information**
None

**Comment by segunadebayo** [comment, marked as answer]
This is v2 Select. Guard `onChange` and skip `""` or the placeholder value.
